api-catalog-ui/frontend/src/components/ServicesNavigationBar/ServicesNavigationBar.jsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+import { Component } from 'react';
+import { Typography } from '@material-ui/core';
+import SideBarLinks from './SideBarLinks';
+import Shield from '../ErrorBoundary/Shield/Shield';
+import SearchCriteria from '../Search/SearchCriteria';
+import './ServicesNavigationBar.css';
+
+export default class ServicesNavigationBar extends Component {
+    componentWillUnmount() {
+        const { clear } = this.props;
+        clear();
+    }
+
+    handleSearch = (value) => {
+        const { filterText } = this.props;
+        filterText(value);
+    };
+
+    render() {
+        const { match, services, searchCriteria, storeCurrentTileId } = this.props;
+        const hasTiles = services && services.length > 0;
+        const hasSearchCriteria = searchCriteria !== undefined && searchCriteria !== null && searchCriteria.length > 0;
```

Comment:
Not sure, but is it better to use `typeof searchCriteria !== undefined`?

Reply:
Probably it could be just `searchCriteria`

Reply:
You're right actually `typeof searchCriteria !== undefined` would also be good to have, I will add it